packages/story-editor/src/components/form/tags/input.js

```diff
@@ -126,6 +161,7 @@ function Input({
         },
         removeTag: (tag) => () => {
           dispatch({ type: ACTIONS.REMOVE_TAG, payload: tag });
+          inputRef?.current.focus();
```

Comment:
🤔 may be able to just use `setIsInputFocused` here instead of adding a new ref? Same with a few other places.

If not then I think we should remove that extra state and only focus using the ref.

Reply:
hrmmm this gets a little weird because the canvas takes focus away without forcing the input back into focus on these events. I was seeing the ref and the state separately since the state controls what the input looks like with the border.

Reply:
maybe we remove the other state then and just use the ref?

Reply:
then the blue border would disappear 😕

Reply:
oh that's right. okay makes sense! Maybe we just add a comment to make sure people know why we're using both in the future?

Reply:
sure!